packages/react/README.md

```diff
@@ -1,26 +1,125 @@
-# JSONForms - More Forms. Less Code
+# JSON Forms - More Forms. Less Code
+
 ### Complex Forms in the blink of an eye
 
-JSONForms eliminates the tedious task of writing fully-featured forms by hand by leveraging the capabilities of JSON, JSON Schema and Javascript.
+JSON Forms eliminates the tedious task of writing fully-featured forms by hand by leveraging the capabilities of JSON, JSON Schema and Javascript.
+
+## React Package
 
-# React Package
-This is the JSONForms react package which provides the necessary bindings for React. It uses [JSONForms Core](https://www.npmjs.com/package/@jsonforms/core).
+This is the JSON Forms React package which provides the necessary bindings for React. It uses [JSON Forms Core](https://www.npmjs.com/package/@jsonforms/core).
 
```

Comment:
Also mention and link the React seed and the React documentation on JSON forms website.